README.md

```diff
@@ -1,15 +1,26 @@
-p-net: PROFINET device stack
-============================
+p-net Profinet stack
```

Comment:
The stack is only for Profinet devices, should we make that clear here?

Reply:
Done